src/providers/ArrayDataProvider.ts

```diff
@@ -29,7 +29,7 @@ class ArrayDataProvider<T> extends DataProviderBase<T, ArrayDataProviderOptions<
 		} = state;
 		let items = data;
 		if (sort && sort.length) {
-			items = items.sort((a: any, b: any) => {
+			items = [ ...items].sort((a: any, b: any) => {
```

Comment:
That's what I thought but I had a test failing and I tried out:

```var alphabet = ['d', 'c', 'b', 'a'];
alphabet.sort();
console.log(alphabet);```

and I get `["a", "b", "c", "d"]`